lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/ActiveContextBridge.java

```diff
@@ -71,6 +71,16 @@ public void submitTaskString(final String taskConfigurationString) {
     ((EvaluatorContext)jactiveContext).submitTask(taskConfigurationString);
   }
 
+  public String getEvaluatorDescriptorString() {
+    final String descriptorString = Utilities.getEvaluatorDescriptorString(jactiveContext.getEvaluatorDescriptor());
+    LOG.log(Level.FINE, "active context - serialized evaluator descriptor: " + descriptorString);
+    return descriptorString;
+  }
+
+  /**
+   * @deprecated in 0.14. Use getEvaluatorDescriptorString instead.
+   */
+  @Deprecated
```

Comment:
When you deprecate method to replace it with the exact same one under different name, please don't duplicate the code. Make the old method name call the new one instead. If we have to change logic before the old method is deleted, the two versions of code can become different